src/sentry/db/models/fields/slug.py

```diff
@@ -0,0 +1,12 @@
+from django.core.exceptions import ValidationError
+from django.db.models import SlugField
+
+
+class NoNumericValidator:
+    def __call__(self, value: str):
```

Comment:
a class with just `__call__` is usually better written as just a function

Reply:
I was just trying to keep the class structure that the other core validators from Django have, and if it needs to update in the future. But fair, it's so simple it should just be a function for now. Changed!